.sage/main.go

```diff
@@ -51,6 +52,16 @@ func GoLint(ctx context.Context) error {
 	return sggolangcilint.Run(ctx)
 }
 
+func GoLintFix(ctx context.Context) error {
+	sg.Logger(ctx).Println("fixing Go files...")
+	return sggolangcilint.Fix(ctx)
+}
+
+func GoFormat(ctx context.Context) error {
+	sg.Logger(ctx).Println("formatting Go files...")
+	return sggolines.Run(ctx)
+}
```

Comment:
Hmmm;

```bash
❯ make go-format     
[sage] building binary and generating Makefiles...
[go-format] formatting Go files...
[sggofumpt:prepare-command] fetching https://github.com/mvdan/gofumpt/releases/download/v0.6.0/gofumpt_v0.6.0
_darwin_arm64 ...
[sggolines:prepare-command] fetching https://github.com/segmentio/golines/releases/download/v0.12.2/golines_0
.12.2_darwin_all.tar.gz ...
[go-format] [2024-03-07 18:26:18] FATAL Error formatting source: exec: "'gofumpt": executable file not found 
in $PATH
[go-format] exit status 1
make: *** [go-format] Error 1
```

Reply:
interesting - that "fix" did look too good to be true!

Reply:
except for in edge cases it should work with `gofumpt` so let's start with that?

Reply:
```bash
❯ make go-format
[sage] building binary and generating Makefiles...
[go-format] formatting Go files...
```

👍